src/pages/instances/forms/DuplicateInstanceForm.tsx

```diff
@@ -64,20 +66,22 @@ const DuplicateInstanceForm: FC<Props> = ({ instance, close }) => {
     queryFn: () => fetchInstances(instance.project),
   });
 
-  const notifySuccess = (instanceName: string, instanceProject: string) => {
+  const notifySuccess = (
+    name: string,
+    project: string,
+    type: InstanceIconType,
+  ) => {
+    const url = `/ui/project/${project}/instance/${name}/configuration`;
     const message = (
       <>
-        Created instance <strong>{instanceName}</strong>.
+        Created instance <ResourceLink type={type} value={name} to={url} />.
```

Comment:
The chip should direct to the overview section. Maybe check other places, there might be more outliers like this one where we reuse the url.

Reply:
I remember this was discussed when we did the duplicate features, since the user can't configure the instance in the duplicate form, we want to create a notification that directs them to the config page. This is also indicated by the "Configure" action associated to the notification. I think that still makes sense and I'm not sure if we should now redirect to overview page?

Reply:
I am fine with the configure link going to the configure page. But I think the chips should always go to the overview page, so the user is confirmed in the mental modal of how the chips work.